packages/pie/src/hooks.ts

```diff
@@ -49,16 +72,19 @@ export const useNormalizedData = ({
                 const datumId = getId(datum)
                 const datumValue = getValue(datum)
 
-                const normalizedDatum = {
+                const normalizedDatum: Omit<ComputedDatum<R>, 'arc' | 'color' | 'fill'> = {
                     id: datumId,
+                    // @ts-ignore
```

Comment:
Is this comment still needed?

Reply:
It was, but I found a cleaner way to handle this.

Reply:
Adjusted to use a nullish operator.